docs/user_guide.rst

```diff
@@ -8,12 +8,29 @@ code snippets and check the output yourself.
 
 .. note::
 
-   At the moment IQM does not provide a quantum computing service open to the general public.
-   Please contact our `sales team <https://www.meetiqm.com/contact-us/>`_ to set up your access to an IQM quantum
-   computer.
+   IQM provides access to its quantum computers via IQM Resonance – IQM's quantum cloud service.
+   Please head over `to our website <https://www.meetiqm.com/products/iqm-resonance/>`_ to learn more.
 
 
-Hello, world!
+Hello, world! (IQM Resonance)
+-------------
+
+Here's the quickest and easiest way to run a small computation on an IQM quantum computer and check that
+things are set up correctly:
+
+1. Login to `IQM Resonance <https://resonance.meetiqm.com>` with your credentials
+2. Generate an API token using the dashboard
```

Comment:
I would suggest adding some suggestions on how to handle the access token, maybe just copy the test from Resonance docs. I can imaging how people generate it in this step, close the popup, then go to step 5 and like "where is the access token?", have seen it in the Hanken hackathon few weeks ago.